Formula/coq.rb

```diff
@@ -36,11 +35,10 @@ def install
     ENV.prepend_path "OCAMLPATH", Formula["ocaml-findlib"].opt_lib/"ocaml"
     system "./configure", "-prefix", prefix,
                           "-mandir", man,
-                          "-docdir", pkgshare/"latex",
-                          "-coqide", "no",
-                          "-with-doc", "no"
-    system "make", "world"
-    ENV.deparallelize { system "make", "install" }
+                          "-docdir", pkgshare/"latex"
+    system "make", "dunestrap"
+    system "dune", "build", "-p", "coq-core,coq-stdlib,coq"
+    system "dune", "install", "--prefix=#{prefix}", "--mandir=#{man}", "coq-core", "coq-stdlib", "coq"
```

Comment:
Seems nicer to add this as a formula so we can keep coqide: https://github.com/garrigue/lablgtk

But I guess we can leave that for someone who uses it themselves.

Reply:
The previous version was built with the flag `"-coqide", "no",` (this flag doesn't exist anymore in `8.17.0`) so I think we never shipped the IDE. But I agree it would be nice to build it, I'll take a look at `lablgtk`.